Sources/CouchDB/CouchDBClient.swift

```diff
@@ -16,7 +16,7 @@
 
 import Foundation
 import SwiftyJSON
-import KituraNet
+import KituraNIO
```

Comment:
All these changes should now be unnecessary (due to https://github.com/IBM-Swift/Kitura-NIO/pull/72)